modules/node_modules/@ncigdc/containers/MostAffectedCasesChart.js

```diff
@@ -0,0 +1,117 @@
+// @flow
+
+import React from 'react';
+import Relay from 'react-relay';
+import withSize from 'react-sizeme';
+import { compose, lifecycle } from 'recompose';
+import withRouter from '@ncigdc/utils/withRouter';
+import { parseFilterParam } from '@ncigdc/utils/uri';
+import SpinnerParticle from '@ncigdc/uikit/Loaders/Particle';
+import { Row } from '@ncigdc/uikit/Flex';
+import BarChart from '@ncigdc/components/Charts/BarChart';
+import { withTheme } from '@ncigdc/theme';
+
+const MostAffectedCasesChartComponent = compose(
+  withTheme,
+  withSize(),
+  withRouter,
+  lifecycle({
+    componentDidMount() {
+      this.props.relay.setVariables({
+        fetchData: true,
+        macChart_filters: parseFilterParam(
+          this.props.query.macChart_filters,
+          this.props.defaultFilters || null
+        ),
+      });
+    },
+  })
+)(({
+  cohort: { cases },
+  theme,
+  size: {
+    width,
+  },
+  push,
+}) => {
+  if (!cases) {
+    return (
+      <Row style={{ justifyContent: 'center', padding: '1rem' }}>
```

Comment:
this was popping up all over the place so I made a component, should work here? https://github.com/NCI-GDC/portal-ui/blob/relay/modules/node_modules/%40ncigdc/components/SpinnerCentered.js